src/main/java/com/google/flourbot/entity/EntityModule.java

```diff
@@ -1,12 +1,9 @@
 package com.google.flourbot.entity;
 
-import java.lang.Exception;
 import java.util.Optional;
-import com.google.flourbot.datastorage.FirebaseDataStorage;
-
-import java.lang.InterruptedException;
 import java.util.concurrent.ExecutionException;
 
 public interface EntityModule {
-    public Optional<Macro> getMacro(String userEmail, String macroName) throws InterruptedException, ExecutionException;
-}
\ No newline at end of file
+  Optional<Macro> getMacro(String userEmail, String macroName)
+      throws InterruptedException, ExecutionException;
```

Comment:
once you fix the FirebaseDataStorage as suggested this will not be necessary.

Reply:
Done!